packages/apps/spaces/app/organizations/[id]/components/Timeline/events/action/RenewalForecastUpdatedAction.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Flex } from '@ui/layout/Flex';
+import { FeaturedIcon, Icons } from '@ui/media/Icon';
+import { getFeatureIconColor } from '@organization/components/Tabs/panels/AccountPanel/utils';
+import { Text } from '@ui/typography/Text';
+import { Action } from '@graphql/types';
+
+const DEFAULT_COLOR_SCHEME = 'gray';
+
+// You may want to tweak this function according to your needs
+const getCurrencyString = (text: string) => {
+  const match = text.split(/(\$[\d,]+(\.\d{2})?)/).filter(Boolean);
+  return match?.[1];
+};
+
+interface RenewalForecastUpdatedActionProps {
+  data: Action;
+}
+
+export const RenewalForecastUpdatedAction: React.FC<
+  RenewalForecastUpdatedActionProps
+> = ({ data }) => {
+  let metadata;
+  try {
+    metadata = data?.metadata && JSON.parse(data.metadata);
+  } catch (error) {
+    console.error('Error parsing metadata', error);
+  }
```

Comment:
I would encapsulate the above code in a iife or a plain function to keep it away from the component scope. Also I think we should not log errors to the console intentionally (I agree that we should track this in sentry though.).